unfoldingWord/translationCore
# Fix Check Selection

#### This pull request addresses:
This PR fixes a function that ensures the check is valid. It was giving false negatives

#### How to test this pull request:
Try opening of checks in tN and tW and ensuring that it shows the right one. If the quote is missing from the verse the check should not show. I think we need a usability feature that dialogs the user when the check is not available


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2396)
